test/User-test.js

```diff
@@ -16,9 +17,13 @@ describe('User', () => {
     let hydrationObject8;
     let hydrationObject9;
     let hydrationObject10;
-    let array;
-    let array1;
-
+    let hydrationArray;
+    let hydrationArray1;
+    let sleepObject1;
+    let sleepObject2;
+    let sleepObject3;
+    let sleepArray;
+    let sleepArray1;
```

Comment:
Today I learned we can lump these declarations together i.e., 
`let hydrationArray, hydrationArray1, sleepObject1, sleepObject2,` etc., we may want to do this to shorten up this code :)

Reply:
I would like that as well; I also thought we could possibly turn those bigger objects (hydration, sleep) into one liners instead of longer chunks of code)